interrupts.c

```diff
@@ -4,28 +4,6 @@
 
 extern const char _ebase[];
 
-void intr_init() {
```

Comment:
I don't like reading code like this aswell so I totally understand your pain. But this has to be moved because tlb_refill_handler isn't visible yet, so intr_init has to be at the end of file.